webviews/components/Collaborators.svelte

```diff
@@ -0,0 +1,54 @@
+<script>
+  import { query } from "svelte-apollo";
+  import * as queries from "./queries.js";
+
+  export let project;
+  export let container;
+
+  console.log(project);
+  console.log(container);
+
+  const collaborators =
+    container.type === "repo"
+      ? query(queries.GET_REPO_COLLABORATORS, {
+          variables: {
+            name: container.name,
+            owner: container.owner.login,
+            number: parseInt(project.number),
+          },
+          pollInterval: 1800,
+        })
+      : container.type === "org"
+      ? query(queries.GET_ORG_MEMBERS, {
+          variables: {
+            login: container.login,
+            number: parseInt(project.number),
+          },
+          pollInterval: 1800,
```

Comment:
maybe we can increase interval by _a lot_?
Since it won't change much?

Reply:
fixed in recent push